bitnami/airflow/templates/config/configmap-pod-template.yaml

```diff
@@ -200,6 +200,9 @@ data:
             - name: empty-dir
               mountPath: /opt/bitnami/airflow/airflow.cfg
               subPath: app-base-dir/airflow.cfg
+            - name: empty-dir
+              mountPath: /opt/bitnami/airflow/config/airflow_local_settings.py
+              subPath: app-conf-dir/airflow_local_settings.py
```

Comment:
One final question, why did you include this change? This is not related to the emptyDir volume.

Reply:
Thanks for the explanation.